test/unit/vec/src/VecTest.cpp

```diff
@@ -280,3 +280,22 @@ TEMPLATE_LIST_TEST_CASE("vecNDConstructionFromNonAlpakaVec", "[vec]", alpaka::te
         REQUIRE(nonAlpakaVec[d] == alpakaVec[d]);
     }
 }
+
+TEST_CASE("structuredBindings", "[vec]")
+{
+    using Dim = alpaka::DimInt<2u>;
+    using Idx = std::size_t;
+    using Vec = alpaka::Vec<Dim, Idx>;
+
+    Vec vec{Idx{1}, Idx{2}};
+
+    const auto [y, x] = vec;
```

Comment:
Remember that we're still on C++14 by default. You'll need to enable C++17 explicitly for the test case. Maybe also create a new test target so that we don't require C++17 for the rest of VecTest.

Reply:
I disabled the test if structured bindings are not supported.